cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -327,6 +327,7 @@ public void run() {
           try {
             ConsumerRecords<byte[], byte[]> consumerRecords = _consumer.poll(10);
             if (consumerRecords == SHUTDOWN_RECORDS) {
+              LOG.warn("Metric loader received empty records");
```

Comment:
This should be `trace` -- i.e. we expect to receive empty records in case no new data is available.